src/Xamarin.Android.Build.Tasks/Tasks/Aapt2.cs

```diff
@@ -166,12 +168,19 @@ protected bool LogAapt2EventsFromOutput (string singleLine, MessageImportance me
 					}
 				}
 
+				bool manifestError = false;
+				if (AndroidManifestFile != null && string.Compare (Path.GetFileName (file), Path.GetFileName (AndroidManifestFile.ItemSpec), StringComparison.OrdinalIgnoreCase) == 0) {
+					manifestError = true;
+				}
+
 				// Strip any "Error:" text from aapt's output
 				if (message.StartsWith ("error: ", StringComparison.InvariantCultureIgnoreCase))
 					message = message.Substring ("error: ".Length);
 
 				if (level.Contains ("error") || (line != 0 && !string.IsNullOrEmpty (file))) {
 					LogCodedError (GetErrorCode (message), message, file, line);
+					if (manifestError)
+						LogCodedError (GetErrorCode (message), message, AndroidManifestFile.ItemSpec, line);
```

Comment:
On the plus side, I believe that the original message comes from `aapt`/etc., and thus we don't need to localize it.

On the downside, should we be emitting the *same* error message for *both* filenames?  Or should we distinguish these messages somehow?

```
obj\Debug\…\AndroidManifest.xml:2: error APT2001: already defined a <flag> …
Properties\AndroidManifest.xml:2: error APT2001: Template file for preceding error message
```

If we do this, we'll need to localize the Template file for preceding error message" text.

Reply:
🤔

Reply:
To throw in my 2 cents, I like the sound of two slightly different messages. If I'm following the PR correctly, I believe there will be cases where there will be errors that only affect the generated version of the file, for example due to a typo in a resource name within an `ActivityAttribute.Label` property. Having a different message for the template file will hopefully make the error messages for that scenario a little less confusing.